specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -792,20 +859,20 @@ paths:
       parameters:
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
```

Comment:
Het gaat niet om "ingeschrevenpersoon", want dat is geen property in de beslag resource. Ik vermoed dat het gaat om "beslaglegger". Dit hebben we nu gemodelleerd als persoon, waarbij type zegt welk type het is (ingeschreven, kadaster natuurlijk, kadaster niet natuurlijk).

Voorstel: beslaglegger__burgerservicenummer.

Kan je dan ook gelijk de description aanpassen, want die maakt niet duidelijk welke persoon (rol) bedoeld wordt.

Reply:
Zie eerdere reactie. beslaglegger is geen resource volgens mij. De resource is de persoon en die persoon heeft de rol beslaglegger. Dit staat wat mij betreft open voor verdere discussie, maar ik voor de voortgang merge ik deze pull request wel.  Ik zal hier een issue voor aanmaken. (#331)